packages/ra-core/src/dataProvider/useInfiniteGetList.ts

```diff
@@ -101,24 +101,31 @@ export const useInfiniteGetList = <RecordType extends RaRecord = any>(
             'getInfiniteList',
             { pagination, sort, filter, meta },
         ],
-        queryFn: ({ pageParam = pagination.page, signal }) =>
```

Comment:
Not sure the changes are necessary here, cf https://github.com/TanStack/query/issues/3633#issuecomment-1133581032:

> with InfiniteQueries, we currently always abort fetches even if the AbortSignal is not consumed, which is likely a bug

Reply:
The *which is likely a bug* makes me think we should do it here just in case